client/src/components/layout/PlanetShow.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react"
+
+const PlanetShow = (props) => {
+  const [planet, setPlanet] = useState({})
+
+  const planetId = props.match.params.id
+  const getPlanet = async () => {
+    try {
+      const response = await fetch(`/api/v1/planets/${planetId}`)
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw (error)
+      }
+      const responseBody = await response.json()
+      setPlanet(responseBody.planet)
+    } catch (err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
+  }
+
+  useEffect(() => {
+    getPlanet()
+  }, [])
+
+  return (
+    <div className="planet-show">
+      <img src={planet.imageUrl} width="100px" className="planet-image" alt="image of planet" />
```

Comment:
avoid doing inline styling e.g. `width`. Instead, add the width to your className of `planet-image`